### PR TITLE
strawberry: 0.9.1 -> 0.9.2

### PR DESCRIPTION
**Pull request by ppenguin**
###### Motivation for this change

update to newest upstream

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 117889` at 6f4a9265 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built successfully:</summary>
  <ul>
    <li>strawberry</li>
  </ul>
</details>
<details>
  <summary>2 suggestions:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/build-tools-in-build-inputs.md">build-tools-in-build-inputs</a>

util-linux is a build tool so it likely goes to `nativeBuildInputs`, not `buildInputs`.

Near pkgs/applications/audio/strawberry/default.nix:47:3:
```
   |
47 |   buildInputs = [
   |   ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/unclear-gpl.md">unclear-gpl</a>

`gpl3` is a deprecated license, please check if project uses `gpl3Plus` or `gpl3Only` and change `meta.license` accordingly.

Near pkgs/applications/audio/strawberry/default.nix:94:5:
```
   |
94 |     license = licenses.gpl3;
   |     ^
```
</li>
  </ul>
</details>

File: pkgs/applications/audio/strawberry/default.nix [hunk 1]
```diff
@@ -77,7 +76,12 @@ mkDerivation rec {
   ])
   ++ lib.optional withVlc libvlc;
 
-  nativeBuildInputs = [ cmake ninja pkg-config qttools ];
+  nativeBuildInputs = [
+    cmake ninja pkg-config qttools
+  ]
+  ++ lib.optionals stdenv.isLinux [
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/strawberry/default.nix:82]:
```suggestion
  ] ++ lib.optionals stdenv.isLinux [
```

File: pkgs/applications/audio/strawberry/default.nix [hunk 2]
```diff
@@ -67,7 +67,6 @@ mkDerivation rec {
     libselinux
     libsepol
     p11-kit
-    util-linux
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/strawberry/default.nix:70]: Please don't blindly trust the bot as the suggestion may be not 100% correct.

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]
Please squash the commits together into one with the message ``strawberry: 0.9.1 -> 0.9.2``.

**Comment by ppenguin** [comment]
> Please squash the commits together into one with the message `strawberry: 0.9.1 -> 0.9.2`.

I attempted this with commit 8c64e76ba2447589246fe9f6ab14957e529dd48d, but not so sure how to get rid of all the other commits...

(I did a `git rebase -i HEAD~x`, since github doesn't apparently have a squash button?)

**Comment by SuperSandro2000** [comment]
> (I did a `git rebase -i HEAD~x`, since github doesn't apparently have a squash button?)

Well it has but it is nicer to not needing to squash them.

**Pull request merged**